tf-modules/data-persistence/README.md

```diff
@@ -0,0 +1,45 @@
+# Data Persistence
+
+This module deploys data persistence resources, including DynamoDB tables and an Elasticsearch instance (which is optional).
+
+## Deployment
+
+1. Copy the .tfvars sample file: `cp terraform.tfvars.sample terraform.tfvars`
+2. Populate the sample file with values that apply to your AWS environment (see configuration variables section below).
+3. Deploy this module: `terraform apply`
+
+NOTE: Terraform will ignore the `aws_profile` config variable if you have static credentials or environment variables set, see the [AWS Provider page](https://www.terraform.io/docs/providers/aws/index.html#authentication).
+
+## Included resources
+
+- DynamoDB tables:
+  - `AccessTokensTable`
+  - `AsyncOperationsTable`
+  - `CollectionsTable`
+  - `ExecutionsTable`
+  - `FilesTable`
+  - `GranulesTable`
+  - `PdrsTable`
+  - `ProvidersTable`
+  - `RulesTable`
+  - `SemaphoresTable`
+  - `UsersTable`
+- Elasticsearch domain (with optional VPC configuration)
```

Comment:
Is it worth noting that we provide auto-scaling via the  DynamoDB On-Demand feature?

Reply:
I'd lean toward "no".

Reply:
I don't have strong feelings. It might be nice to mention that the billing mode is `PAY_PER_REQUEST`, but then do we get into the territory of documenting all the other defaults?